evasquez26/angular4-startup
# S1 one 90 settings button

## Story / Task

[ONE-90|Story](https://bixlabs.atlassian.net/browse/ONE-90)

## What this PR does?

Adds the settings button on the navbar.

### Before 

We did not have a settings button

### After

- We have a settings button when clicking on the user name
- Clicking on settings button redirects to the settings view

### How to test / reproduce

- Run the app.
- Observe that when you click the user name a dropdown menu appears
- Clicking the settings button redirects to settings view

### TODOs
- [x] Rebase after merging #4 

### Related PRs

N/A

## Migrations

NO

Thanks!
